client/src/components/DashboardAppBar/useStyles.ts

```diff
@@ -0,0 +1,21 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+const primaryColor = '#759CFC';
```

Comment:
we should be putting this in the theme rather than just a constant so it can be used widely acrosss the app